doc/classes/OccluderInstance3D.xml

```diff
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="OccluderInstance3D" inherits="Node3D" version="4.0">
 	<brief_description>
+		Provides occlusion culling for 3D nodes, which improves performance in closed areas.
 	</brief_description>
 	<description>
+		Occlusion culling can improve rendering performance in closed/semi-open areas by hiding geometry that is occluded by other objects.
+		The occlusion culling system is mostly static. [OccluderInstance3D]s can be moved or hidden at run-time, but doing so will trigger a background recomputation that can take several frames. It is recommended to only move [OccluderInstance3D]s sporadically (e.g. for procedural generation purposes), rather than doing so every frame.
+		The occlusion culling system works by rendering the occluders on the CPU in parallel using [url=https://www.embree.org/]Embree[/url], drawing the result to a 160×90 buffer then using this to cull 3D nodes individually. In the 3D editor, you can preview the occlusion culling buffer by choosing [b]Perspective &gt; Debug Advanced... &gt; Occlusion Culling Buffer[/b] in the top-left corner of the 3D viewport.
```

Comment:
The occlusion buffer size is not fixed, it depends on the "rendering/occlusion_culling/occlusion_rays_per_thread" project setting (I now see this may not be a very explicit setting name, we could change it). The resolution is chosen so the number of pixels in the occlusion buffer is approximately (rays_per_thread * cpu_physical_threads). This means CPUs with fewer cores will use a lower resolution, in an attempt to keep performance costs more even across devices.